DcCore/DcCore/DC/Wrapper.swift

```diff
@@ -456,6 +480,21 @@ public class DcContext {
         return swiftString
     }
 
+    public func getMessageReactions(messageId: Int) -> DcReactions? {
+        if let data = dcAccounts.blockingCall(method: "get_message_reactions", params: "\(id), \(messageId)") {
+            return try? JSONDecoder().decode(DcReactionResult.self, from: data).result
+        }
+        return nil
+    }
+
+    public func sendReaction(messageId: Int, reaction: String?) {
+        if let reaction = reaction {
```

Comment:
```suggestion
        if let reaction {
```

The `= reaction` is not needed anymore since [Swift 5.7](https://www.swift.org/blog/swift-5.7-released/)